test/tests/outputs/format/test_json.py

```diff
@@ -11,8 +11,8 @@ def find_app():
     """
     moose_dir = os.environ.get("MOOSE_DIR")
     if not moose_dir:
-        file_dir = os.dirname(os.path.abspath(__file__))
-        moose_dir = os.dirname(os.dirname(file_dir))
+        file_dir = os.path.dirname(os.path.abspath(__file__))
+        moose_dir = os.path.abspath(os.path.join(file_dir, "..", "..", "..", ".."))
```

Comment:
I think a better approach is to use git to tell us:
https://github.com/idaholab/moose/blob/devel/framework/scripts/find_dep_apps.py#L17